x/ibc/02-client/alias.go

```diff
@@ -48,8 +46,6 @@ var (
 )
 
 type (
-	Keeper          = keeper.Keeper
-	StakingKeeper   = types.StakingKeeper
-	MsgCreateClient = types.MsgCreateClient
```

Comment:
Do we not have these txs here anymore?

Reply:
No the interfaces are in `exported`, so my understanding is they shouldn't be aliased here to avoid import cycles